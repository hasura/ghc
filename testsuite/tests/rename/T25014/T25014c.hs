-- Should not compile trying to export a missing name
module T25014c (A (foo)) where

data A

data B = B {
  foo :: Int
}
