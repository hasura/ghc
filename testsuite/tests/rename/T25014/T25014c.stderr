T25014c.hs:2:17: [GHC-88993]
     The type constructor ‘A’ is not the parent of the record selector ‘foo’.
      Record selectors can only be exported with their parent type constructor.
      Parent: B
     In the export: A(foo)